B827EBFFFEBFB1AF.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "B827EBFFFEBFB1AF",
+    "servers": [
+      {
+        "server_address": "eu.thethings.network",
```

Comment:
The server address is not valid. Please check [this list](https://www.thethingsnetwork.org/wiki/Backend/Connect/Gateway).